nima_core/api/server.py

```
@@ -313,6 +315,49 @@ async def health() -> Dict[str, Any]:
+                "turn_id": row["turn_id"] or "",
+                "timestamp": row["timestamp"] or 0,
+                "who": row["who"] or "unknown",
+                "layer": row["layer"] or "episodic",
+                "summary": row["summary"] or "",
+                "text": row["text"] or "",
+                "score": score_map.get(mid, 0.0),
+                "strength": row["strength"] if row["strength"] is not None else 1.0,
+                "is_ghost": bool(row["is_ghost"]) if row["is_ghost"] is not None else False,
+            })
+        return results
+    except Exception as exc:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)